hw/top_earlgrey/data/ip/chip_pwrmgr_testplan.hjson

```diff
@@ -69,7 +69,7 @@
         "PWRMGR.LOW_POWER.WAKE_INFO"
       ]
       tests: ["chip_sw_pwrmgr_random_sleep_all_wake_ups"]
-      bazel: []
+      bazel: ["//sw/device/tests:pwrmgr_random_sleep_all_wake_ups_fpga_cw310_test_rom"]
```

Comment:
Random / off-topic: Is this testplan file actually supposed to be geared towards the FPGA? It seems a bit peculiar to call out the CW310's execution environment explicitly. I'm not sure how this file is actually used, though--If it's merely documentation, then perhaps it doesn't matter.

Reply:
It is really an indication that a non-sim-dv test is running, which is a strong hint that it may not be much more work for it to be silicon ready.

Incidentally, do we have a bazel target that corresponds to a silicon test? Even if we did, we would not be able to run it.